packages/otelbin/src/components/monaco-editor/ValidationErrorConsole.tsx

```diff
@@ -195,10 +189,10 @@ export function ErrorMessage({
 					<p className="max-w-[100%]">{`${customErrors}`}</p>
 				</div>
 			)}
-			{jsYamlError ? (
+			{yamlError ? (
 				<div className={`${font.className} ${errorsStyle}`}>
-					<p className="max-w-[100%]">{`${jsYamlError.reason} ${
-						jsYamlError.mark && `(Line ${jsYamlError.mark.line})`
+					<p className="max-w-[100%]">{`${yamlError.code} ${
```

Comment:
**Question:** Why are we using the short version here? IMHO that is hard to understand, and we seem have enough room in the console for the long version?

Reply:
Sure, it will be change to the long version

Reply:
The long version message is pushed and committed.